page-controller/README.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: pattern
+title: Page Controller
+folder: parameter-controller
+permalink: /patterns/parameter-object/
+categories: Structural
+language: en
+tags:
+- Decoupling
+---
```

Comment:
Check out the yaml frontmatter requirements at https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute